octoprint_PrintTimeGenius/analyzers/analyze_gcode_comments.py

```diff
@@ -6,10 +6,12 @@
 import sys
 from collections import defaultdict
 import argparse
-if float('.'.join((str(sys.version_info.major), str(sys.version_info.minor)))) < 3.3:
-  from collections import Mapping as collections_Mapping
```

Comment:
Ah shit, this is bad.  This was a PR that I accepted and I should have pushed back because it's not done right.  :-(  Sorry about that.